src/agent/custom_agent.py

```
@@ -22,19 +22,23 @@
 )
 from browser_use.utils import time_execution_async
 from langchain_core.language_models.chat_models import BaseChatModel
 from langchain_core.messages import (
     BaseMessage,
+    HumanMessage,
+    AIMessage
 )
+from browser_use.agent.prompts import PlannerPrompt
+
 from json_repair import repair_json
 from src.utils.agent_state import AgentState
```

Ruff (0.8.2)

`src.utils.agent_state.AgentState` imported but unused

Remove unused import: `src.utils.agent_state.AgentState`

(F401)